### PR TITLE
Vector Index updates

### PR DESCRIPTION
**Pull request by claireyuw**
fix: #385

### CODE REVIEW

File: site/en/reference/index.md [hunk 1]
```diff
@@ -85,210 +33,9 @@ Milvus stores massive data in sections. When indexing, Milvus creates an index f
...
-   | Parameter   | Description     | Range     |
-   | -----------|--------------------------------- | ---------------- |
-   | `search_k` | The number of nodes to search. -1 means 5% of the whole data. | {-1} ∪ [`top_k`, n × `n_trees`] |
-
 ## How to choose an index
 
-To learn how to choose an appropriate index for your application scenarios, please read [How to Select an Index in Milvus](https://medium.com/@milvusio/how-to-choose-an-index-in-milvus-4f3d15259212).
+To learn about the index types supported by Milvus and how to choose an appropriate index for your application scenarios, please read [How to Select an Index in Milvus](index_selection.md).
 
 To learn how to choose an appropriate index for a metric, see [Similarity Metrics](metric.md).
```
Comment (LocoRichard) [inline site/en/reference/index.md:40]: Is it the other way around? ...choose an appropriate metric for an index?

File: site/en/reference/index.md [hunk 2]
```diff
@@ -85,210 +33,9 @@ Milvus stores massive data in sections. When indexing, Milvus creates an index f
 
 It is known that indexing is a resource-consuming and time-consuming task. When the query task and indexing task are concurrent, Milvus preferentially allocates computing resources to the query task, that is, any query command will interrupt the indexing task being executed in the background. After that, only when the user does not send the query task for 5 seconds, Milvus resumes the indexing task in the background. Besides, if the data segment specified by the query command has not been built into the specified index, Milvus will do an exhaustive search directly within the segment.
```
Comment (LocoRichard) [inline site/en/reference/index.md:34]: delete the space in the front

File: site/en/reference/index.md [hunk 3]
```diff
@@ -1,82 +1,30 @@
...
 
 <div class="alert note">
-When the inserted data segment is less than 4096 rows, Milvus does not index it.
+When the inserted data segment is less than 4,096 rows, Milvus does not index it.
 </div>
 
+
 ### Index by segment
 
 Milvus stores massive data in sections. When indexing, Milvus creates an index for each data segment separately.
```
Comment (LocoRichard) [inline site/en/reference/index.md:30]: ...data in *segments*?

File: site/en/reference/index.md [hunk 4]
```diff
@@ -1,82 +1,30 @@
...
-    <td>Tree-based index</td>
-    <td></td>
-  </tr>
-</tbody>
-</table>
 
 ## Vector field and index
 
-To improve query performance, you can specify an index type for each vector field. Currently, a vector field only supports one index type, Milvus will automatically delete the old index when switching the index type.
+To improve query performance, you can specify an [index type](index_selection.md) for each vector field. Currently, a vector field only supports one index type, Milvus will automatically delete the old index when switching the index type.
```
Comment (LocoRichard) [inline site/en/reference/index.md:17]: one index type. Milvus automatically deletes the old index when switching the index type.

File: site/en/reference/index.md [hunk 5]
```diff
@@ -1,82 +1,30 @@
 ---
 id: index.md
+related_key: index
 ---
 
 # Vector Index
 
-Vector index is a time- and space-efficient data structure built on vectors through a certain mathematical model. Through the vector index, we can efficiently query several vectors similar to the target vector.
+Indexing, a process of oragnizing data, is a huge component of what makes it possible to efficiently query the million-, billion-, or even trillion-vector datasets that vector databases rely on.
```
Comment (LocoRichard) [inline site/en/reference/index.md:8]: what do you mean by `rely on`?

File: site/en/reference/index.md [hunk 6]
```diff
@@ -1,82 +1,30 @@
...
 
-Since accurate retrieval is usually very time-consuming, most of the vector index types of Milvus use ANNS (Approximate Nearest Neighbors Search). Compared with accurate retrieval, the core idea of ANNS is no longer limited to returning the most accurate result, but only searching for neighbors of the target. ANNS improves retrieval efficiency by sacrificing accuracy within an acceptable range.
+## **How does vector indexing work?**
 
-According to the implementation methods, the ANNS vector index can be divided into four categories:
+Similarity search engines work by comparing an input to a database to find objects that are most similar to the input. Indexing is the process of efficiently organizing data, and it plays a major role in making similarity search useful by dramatically accelerating time-consuming queries on large datasets. After a massive vector dataset is indexed, queries can be routed to clusters, or subsets of data, that are most likely to contain vectors similar to an input query. In practice, this means a certain degree of accuracy is sacrificed to speed up queries on really big vector data.
```
Comment (LocoRichard) [inline site/en/reference/index.md:12]: - ...by comparing an input to the objects in a database to find those that
- ...really a big vector dataset.

File: site/zh-CN/reference/index_selection.md [hunk 7]
```diff
@@ -0,0 +1,301 @@
+---
```
Comment (LocoRichard) [inline site/zh-CN/reference/index_selection.md:1]: I think we have the CN version of this page?

**Review by LocoRichard — CHANGES_REQUESTED** [review]

File: site/zh-CN/reference/index_selection.md (hunk 7 above)
Comment (claireyuw) [reply site/zh-CN/reference/index_selection.md:1]: We do, but it needs some revision. I'll update it tomorrow.

**Pull request merged**